src/generatedSheetFunctions.js

```diff
@@ -1459,3 +1485,28 @@ function SAN_SOCIAL_VOLUME_AGGREGATED (projectSlug, from, to, aggregation = 'nul
   )
 }
 
+
+/**
+* Returns the marketcap for a slug.
```

Comment:
Should we mention that it is aggregated?

Reply:
What is the difference between aggregated and the other one?

Reply:
Aggregated returns the timeseries, but aggregated down to a single number.